<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QualityCurve</class>
 <widget class="QWidget" name="QualityCurve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1707</width>
    <height>764</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1707</width>
    <height>764</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1707</width>
    <height>764</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCustomPlot" name="customPlot" native="true">
   <property name="geometry">
    <rect>
     <x>336</x>
     <y>2</y>
     <width>1361</width>
     <height>399</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1531</width>
     <height>399</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget_curve
{
background: #FFFFFF;
border-radius: 10px;
}</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_time_select" native="true">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>410</y>
     <width>1370</width>
     <height>90</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1420</width>
     <height>90</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget_time_select
{
background: #FFFFFF;
border-radius: 10px;
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>PingFang SC</family>
        <pointsize>-1</pointsize>
        <weight>62</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 85px;
height: 22px;
font-family: PingFang SC;
font-weight: 500;
font-size: 22px;
color: #999999;
line-height: 42px;</string>
      </property>
      <property name="text">
       <string>时间范围</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dateEdit_start">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QDateEdit#dateEdit_start {
    background: #FFFFFF;
    border-radius: 30px;
    border: 1px solid #CACDD3;

    font-family: PingFang SC;
    font-weight: 500;
    font-size: 22px;
    color: #333333;
    line-height: 42px;
}
#dateEdit_start::drop-down{

height:55px;
width:50px;
image: url(:/beijing/down_arrow.png);
}
#dateEdit_start::down-button
{
height:55px;
width:45px;
image: url(:/beijing/down_arrow.png);
}

QDateEdit::up-button {
    width: 0px; 
    height: 0px; 
}
QCalendarWidget {
border-radius:10px;
}

QCalendarWidget QWidget#qt_calendar_navigationbar {

	background-color: qlineargradient(spread:pad, x1:0.989, y1:0.949, x2:0.04, y2:0.05, stop:0 rgba(41, 148, 238, 255), stop:1 rgba(45, 204, 233, 255));
border-top-left-radius:10px;
border-top-right-radius:10px;

}

QCalendarWidget QToolButton#qt_calendar_monthbutton:hover {
    color: #343536;
}

QCalendarWidget QToolButton#qt_calendar_yearbutton:hover {
    color: #343536;
}

QCalendarWidget QToolButton#qt_calendar_calendarview {
    background-color: #ff4588;
}

QCalendarWidget QToolButton {
    background-color: transparent;
    border: none;
    border-radius: 10px;
    min-width: 60px;
    min-height: 60px;
    width: 120px;
    height: 60px;
    color: #666666;
    font-size: 24px;
    icon-size: 50px, 50px;
}

QCalendarWidget QToolButton#qt_calendar_prevmonth {
    background-color: transparent;
    width: 62px;
    height: 33px;
    icon-size: 62px, 66px;
	qproperty-icon: url(:/3-1/first_page.png);

}
QCalendarWidget QToolButton#qt_calendar_prevmonth:hover,
QCalendarWidget QToolButton#qt_calendar_prevmonth:pressed {
       icon-size: 62px, 66px;
	   qproperty-icon: url(:/3-1-2/first.png);
}

QCalendarWidget QToolButton#qt_calendar_nextmonth {
    background-color:transparent;
    width: 62px;
    height: 66px;
    icon-size: 62px, 66px;
    qproperty-icon:url(:/3-1-2/last.png);
}
QCalendarWidget QToolButton#qt_calendar_nextmonth:hover,
QCalendarWidget QToolButton#qt_calendar_nextmonth:pressed{
    qproperty-icon:url(:/3-1/last_page.png);
}

QCalendarWidget QMenu {
    width: 120px;
    left: 20px;
    color: #666666;
    font-size: 18px;
    background-color: #f1f2f3;
}

QCalendarWidget QSpinBox {
    width: 120px;
    min-width: 120px;
    max-width: 120px;
    font-size: 28px;
    color: #666666;
    background-color: rgb(242, 247, 251);
    selection-background-color: rgb(136, 136, 136);
    selection-color: rgb(255, 255, 255);
}

QCalendarWidget QSpinBox::up-button {
    subcontrol-origin: border;
    subcontrol-position: top right;
    width: 30px;
    height: 28px;
	image: url(:/1-3/up.png);

}

QCalendarWidget QSpinBox::up-button:hover {
    subcontrol-origin: border;
    subcontrol-position: top right;

    width: 30px;
    height: 28px;
	image: url(:/1-3/up.png);
}

QCalendarWidget  QSpinBox::down-button {
    subcontrol-origin: border;
    subcontrol-position:right bottom ;
    width: 30px;
    height: 28px;
    
	image: url(:/1-3/down.png);

}

QCalendarWidget QSpinBox::down-button:hover {
    subcontrol-origin: border;
    subcontrol-position: bottom right;

    width: 30px;
    height: 28px;
    image: url(:/1-3/down.png);
}

QCalendarWidget QSpinBox::up-arrow {
    width: 28px;
    height: 28px;
}

QCalendarWidget QSpinBox::down-arrow {
    width: 28px;
    height: 28px;
}

/* 星期几 */
QCalendarWidget QWidget {
    alternate-background-color: rgb(237, 240, 244);
}

/* 属于当前月份日期 */
QCalendarWidget QAbstractItemView:enabled {
    font-size: 22px;
    color: rgb(66, 66, 66);
    background-color: rgb(237,240,244);
    selection-background-color: rgb(128, 128, 128);
    selection-color: rgb(0, 225, 0);
}

/* 非当前月份的日期 */
QCalendarWidget QAbstractItemView:!enabled {
    font-size: 22px;
    color: rgb(168, 168, 168);
    background-color: rgb(237, 240, 244);
}
</string>
      </property>
      <property name="specialValueText">
       <string/>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2000</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="minimumDate">
       <date>
        <year>2000</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>PingFang SC</family>
        <pointsize>-1</pointsize>
        <weight>62</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family: PingFang SC;
font-weight: 500;
font-size: 22px;
color: #333333;
line-height: 42px;</string>
      </property>
      <property name="text">
       <string>~</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QDateEdit" name="dateEdit_end">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QDateEdit#dateEdit_end {
    background: #FFFFFF;
    border-radius: 30px;
    border: 1px solid #CACDD3;

    font-family: PingFang SC;
    font-weight: 500;
    font-size: 22px;
    color: #333333;
    line-height: 42px;
}
QDateEdit::down-button {
    image: url(:/beijing/down_arrow.png);
    width: 55px;
    height: 50px;
}
#dateEdit_end::drop-down{

height:55px;
width:50px;
image: url(:/beijing/down_arrow.png);
}
QDateEdit::up-button {
    width: 0px; 
    height: 0px; 
}
QCalendarWidget {
border-radius:10px;
}

QCalendarWidget QWidget#qt_calendar_navigationbar {

	background-color: qlineargradient(spread:pad, x1:0.989, y1:0.949, x2:0.04, y2:0.05, stop:0 rgba(41, 148, 238, 255), stop:1 rgba(45, 204, 233, 255));
border-top-left-radius:10px;
border-top-right-radius:10px;

}

QCalendarWidget QToolButton#qt_calendar_monthbutton:hover {
    color: #343536;
}

QCalendarWidget QToolButton#qt_calendar_yearbutton:hover {
    color: #343536;
}

QCalendarWidget QToolButton#qt_calendar_calendarview {
    background-color: #ff4588;
}

QCalendarWidget QToolButton {
    background-color: transparent;
    border: none;
    border-radius: 10px;
    min-width: 60px;
    min-height: 60px;
    width: 120px;
    height: 60px;
    color: #666666;
    font-size: 24px;
    icon-size: 50px, 50px;
}

QCalendarWidget QToolButton#qt_calendar_prevmonth {
    background-color: transparent;
    width: 62px;
    height: 33px;
    icon-size: 62px, 66px;
	qproperty-icon: url(:/3-1/first_page.png);

}
QCalendarWidget QToolButton#qt_calendar_prevmonth:hover,
QCalendarWidget QToolButton#qt_calendar_prevmonth:pressed {
       icon-size: 62px, 66px;
	   qproperty-icon: url(:/3-1-2/first.png);
}

QCalendarWidget QToolButton#qt_calendar_nextmonth {
    background-color:transparent;
    width: 62px;
    height: 66px;
    icon-size: 62px, 66px;
    qproperty-icon:url(:/3-1-2/last.png);
}
QCalendarWidget QToolButton#qt_calendar_nextmonth:hover,
QCalendarWidget QToolButton#qt_calendar_nextmonth:pressed{
    qproperty-icon:url(:/3-1/last_page.png);
}

QCalendarWidget QMenu {
    width: 120px;
    left: 20px;
    color: #666666;
    font-size: 18px;
    background-color: #f1f2f3;
}

QCalendarWidget QSpinBox {
    width: 120px;
    min-width: 120px;
    max-width: 120px;
    font-size: 28px;
    color: #666666;
    background-color: rgb(242, 247, 251);
    selection-background-color: rgb(136, 136, 136);
    selection-color: rgb(255, 255, 255);
}

QCalendarWidget QSpinBox::up-button {
    subcontrol-origin: border;
    subcontrol-position: top right;
    width: 30px;
    height: 28px;
	image: url(:/1-3/up.png);

}

QCalendarWidget QSpinBox::up-button:hover {
    subcontrol-origin: border;
    subcontrol-position: top right;

    width: 30px;
    height: 28px;
	image: url(:/1-3/up.png);
}

QCalendarWidget  QSpinBox::down-button {
    subcontrol-origin: border;
    subcontrol-position:right bottom ;
    width: 30px;
    height: 28px;
    
	image: url(:/1-3/down.png);

}

QCalendarWidget QSpinBox::down-button:hover {
    subcontrol-origin: border;
    subcontrol-position: bottom right;

    width: 30px;
    height: 28px;
    image: url(:/1-3/down.png);
}

QCalendarWidget QSpinBox::up-arrow {
    width: 28px;
    height: 28px;
}

QCalendarWidget QSpinBox::down-arrow {
    width: 28px;
    height: 28px;
}

/* 星期几 */
QCalendarWidget QWidget {
    alternate-background-color: rgb(237, 240, 244);
}

/* 属于当前月份日期 */
QCalendarWidget QAbstractItemView:enabled {
    font-size: 22px;
    color: rgb(66, 66, 66);
    background-color: rgb(237,240,244);
    selection-background-color: rgb(128, 128, 128);
    selection-color: rgb(0, 225, 0);
}

/* 非当前月份的日期 */
QCalendarWidget QAbstractItemView:!enabled {
    font-size: 22px;
    color: rgb(168, 168, 168);
    background-color: rgb(237, 240, 244);
}
</string>
      </property>
      <property name="wrapping">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="specialValueText">
       <string/>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2000</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="maximumDate">
       <date>
        <year>9999</year>
        <month>12</month>
        <day>31</day>
       </date>
      </property>
      <property name="minimumDate">
       <date>
        <year>2000</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_searchDate">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">    background: #FFFFFF;
    border-radius: 12px;
    border: 1px solid #CACDD3;

    font-family: PingFang SC;
    font-weight: 500;
    font-size: 22px;
    color: #333333;
    line-height: 42px;</string>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>501</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_project">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>114</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>2000</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>PingFang SC</family>
        <pointsize>-1</pointsize>
        <weight>62</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 43px;
height: 21px;
font-family: PingFang SC;
font-weight: 500;
font-size: 22px;
color: #999999;
line-height: 42px;</string>
      </property>
      <property name="text">
       <string>质控项目</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_project_val">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 43px;
height: 21px;
font-family: PingFang SC;
font-weight: 500;
font-size: 22px;
color: #999999;
line-height: 42px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>100</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_bottom" native="true">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>510</y>
     <width>1371</width>
     <height>241</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>11111</width>
     <height>11111</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget_bottom
{
border-radius: 10px;
background: #FFFFFF;
}

</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>431</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_name">
       <property name="styleSheet">
        <string notr="true">width: 150px;
background: #FFFFFF;
height: 21px;
font-family: PingFang SC;
font-weight: 500;
font-size: 22px;
color: #999999;
line-height: 42px;</string>
       </property>
       <property name="text">
        <string>质控项目</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_name_val">
       <property name="styleSheet">
        <string notr="true">width: 150px;
height: 21px;
font-family: PingFang SC;
font-weight: 500;
font-size: 22px;
color: #999999;
line-height: 42px;
background: #FFFFFF;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_target">
       <property name="styleSheet">
        <string notr="true">width: 150px;
background: #FFFFFF;
height: 21px;
font-family: PingFang SC;
font-weight: 500;
font-size: 22px;
color: #999999;
line-height: 42px;</string>
       </property>
       <property name="text">
        <string>靶值</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_target_val">
       <property name="styleSheet">
        <string notr="true">width: 150px;
height: 21px;
font-family: PingFang SC;
font-weight: 500;
font-size: 22px;
color: #999999;
line-height: 42px;
background: #FFFFFF;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_targetRange">
       <property name="styleSheet">
        <string notr="true">width: 150px;
background: #FFFFFF;
height: 21px;
font-family: PingFang SC;
font-weight: 500;
font-size: 22px;
color: #999999;
line-height: 42px;</string>
       </property>
       <property name="text">
        <string>靶值范围</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_targetRange_val">
       <property name="styleSheet">
        <string notr="true">width: 150px;
height: 21px;
font-family: PingFang SC;
font-weight: 500;
font-size: 22px;
color: #999999;
line-height: 42px;
background: #FFFFFF;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_SD">
       <property name="styleSheet">
        <string notr="true">width: 150px;
background: #FFFFFF;
height: 21px;
font-family: PingFang SC;
font-weight: 500;
font-size: 22px;
color: #999999;
line-height: 42px;</string>
       </property>
       <property name="text">
        <string>SD值</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_SD_val">
       <property name="styleSheet">
        <string notr="true">width: 150px;
height: 21px;
font-family: PingFang SC;
font-weight: 500;
font-size: 22px;
color: #999999;
line-height: 42px;
background: #FFFFFF;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>741</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget#tableWidget
{
background: #FFFFFF;
font-family: PingFang SC;
font-weight: 500;
font-size: 24px;
border-radius: 10px;
color: #333333;
line-height: 42px;
}
QTableWidget::item {
    text-align: center;
}
QHeaderView {
    font-family: PingFang SC;
	font-weight: 500;
	font-size: 22px;
	color: #6D7E92;
	line-height: 42px;
}

QHeaderView::section {
    background-color: #EFF0F2;
	border-top: 1px solid #E1EBF6;
	border-left: none;
	border-bottom: 1px solid #E1EBF6;
	border-right: none;
	line-height: 42px;
}
QTableWidget::item:selected {
    background: #D4F2F5;
	border-top: none;
	border-left: none;
    border-bottom: 1px solid #22A4B3;
	border-right: none;
}</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
